contracts/utils/EventCreation.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+contract SimpleTransaction {
+    address public owner;
```

Comment:
consider inheriting from [openzeppelin's Ownable](https://docs.openzeppelin.com/contracts/2.x/access-control#ownership-and-ownable) & use `onlyOwner` modifier for sending

Reply:
Will do!